From private repo

v-alje opened the pull request.

rmca14 commented:
Thank you for your contribution. Closing this pull request unmerged, because this change was addressed in another pull request in the private repo.

heatherbshapiro committed:
Create create-cosmosdb-flaskmongo.md

CosmosDB Flask-Mongo Example

heatherbshapiro committed:
Update and rename create-cosmosdb-flaskmongo.md to create-mongodb-flask.db

heatherbshapiro committed:
Made requested changes

heatherbshapiro committed:
Update create-mongodb-flask.md

heatherbshapiro committed:
updated TOC.yml

heatherbshapiro committed:
Update create-mongodb-flask.md

heatherbshapiro committed:
updated indents

heatherbshapiro committed:
Update create-mongodb-flask.md

heatherbshapiro committed:
updated Azure sample

heatherbshapiro committed:
Merge branch 'patch-2' of https://github.com/heatherbshapiro/azure-docs-pr into patch-2

heatherbshapiro committed:
updated readme

heatherbshapiro committed:
updated readme

Commit:
initial draft

Commit:
minor edits

Commit:
initial add routes to node

Commit:
title change, link fix

Commit:
revision per change in title recommendation

Commit:
added pricing note alternative

Commit:
fixed numbered steps

Commit:
initial changes per Ashita Rastogi

Commit:
restructuring, minor blurb edits

Commit:
more blurb edits

Commit:
storage portion restructured

Commit:
removed note

Commit:
removed images

Commit:
[Azure Functions] adding scale best practices

Commit:
[Azure Functions] adding scale best practices

SHEETHALJS committed:
(ForumsTeam) "users" word removed

alkohli committed:
removed Classic node for 8000 series

mumian committed:
remove preview

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
migrated changes to .NET and Java articles

Commit:
minor fix

Commit:
another fix

daveba committed:
updating articleactive-directory-coreapps-assign-user-azure-portal.md

daveba committed:
updating spacing

daveba committed:
improving acrolinx score

neilpeterson committed:
initial file

neilpeterson committed:
updated image links

neilpeterson committed:
updated image links

daveba committed:
formatting changes

daveba committed:
formatting changes

Commit:
Update billing-add-change-azure-subscription-administrator.md

genlin committed:
test

Commit:
Merge pull request #3 from genlin/patch-2

test

genlin committed:
Update billing-add-change-azure-subscription-administrator.md

Commit:
Merge pull request #4 from genlin/patch-4

Update billing-add-change-azure-subscription-administrator.md

neilpeterson committed:
updated images

cephalin committed:
use cert article

cephalin committed:
acrolinx

Commit:
branded meta data per rjagiewich

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

MikeDodaro committed:
Image endpoints summary page

MikeDodaro committed:
Added endpoint page to TOC

MightyPen committed:
Rebecca A. suggested this, in earlier PR 28938.

Moving 'Next Steps' down to end.
(I will Close junk PR 28963.)
MightyPen = genemi.

neilpeterson committed:
updates

neilpeterson committed:
updates

MikeDodaro committed:
retry after failed review build.

daveba committed:
updating based on user feedback

neilpeterson committed:
acrolynx and toc

Commit:
redoing new deployment topic

MikeDodaro committed:
Page line-spacing

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

MikeDodaro committed:
Line spacing again

Commit:
stuck on checking exp accts

MikeDodaro committed:
Include description of POST request with image binary data

SnehaGunda committed:
Adding offline syndication content

MikeDodaro committed:
Improve text

SnehaGunda committed:
Monir updates

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

MikeDodaro committed:
Improvements

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

SnehaGunda committed:
formatting list

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

SnehaGunda committed:
formatting list

nitinme committed:
Adding a new article

SnehaGunda committed:
formatting list

neilpeterson committed:
updates

MikeDodaro committed:
Formatting

tfitzmac committed:
add example templates

heatherbshapiro committed:
made markdown changes

Commit:
Correcting the definition of errors

MikeDodaro committed:
Link text fixed

nitinme committed:
fixing validation issues

neilpeterson committed:
resize images

motanv committed:
RDBug 10887895 : [Doc] 6.1 ChaosTargetFilter type is updated in acom document before releasing 6.1

MikeDodaro committed:
More improvements

spelluru committed:
Windows PowerShell -> Azure PowerShell

Commit:
correction in cluster setting doc.

motanv committed:
Merge branch 'master' into remove_target_filter_until_relase

Commit:
Merge pull request #28605 from mumian/domain-joined1127

remove preview

Commit:
fixed typo per ktoliver

Juliako committed:
Update media-services-overview.md

GitHubber17 committed:
Merge pull request #28982 from bradsev/ml-2-deploy

redoing new deployment topic

Commit:
Merge pull request #29016 from v-payc/correctioninSettingdoc

correction in cluster setting doc.

ktoliver committed:
Merge pull request #28996 from SnehaGunda/offline

Adding offline syndication content

GitHubber17 committed:
Merge pull request #29005 from tfitzmac/1130key

add example templates

Commit:
Merge pull request #29017 from Juliako/patch-1

Update media-services-overview.md

ktoliver committed:
Merge pull request #29004 from nitinme/nitinme-freshness-updates

Updates to HDInsight Spark

ashannon7 committed:
Update time-series-insights-explorer.md

MGoedtel committed:
updated supported OS's

GitHubber17 committed:
Merge pull request #28961 from msebolt/iot-hub-node-node-process-d2c-new

Iot hub node node process d2c new

rmca14 committed:
Merge pull request #24587 from heatherbshapiro/patch-2

Create create-cosmosdb-flaskmongo.md

PRMerger17 committed:
Merge pull request #29020 from ashannon7/patch-1

Update time-series-insights-explorer.md

Commit:
Merge pull request #29014 from motanv/remove_target_filter_until_relase

RDBug 10887895 : [Doc] 6.1 ChaosTargetFilter type is updated in acom …

sanjeev3 committed:
Update try-image-api.md

MGoedtel committed:
updated statement to improve readability

sanjeev3 committed:
Update try-image-list-api.md

sanjeev3 committed:
Update try-review-api-job.md

sanjeev3 committed:
Update try-image-list-api.md

sanjeev3 committed:
Update try-review-api-review.md

sanjeev3 committed:
Update try-review-api-workflow.md

sanjeev3 committed:
Update try-terms-list-api.md

sanjeev3 committed:
Update try-review-api-workflow.md

sanjeev3 committed:
Update try-review-api-review.md

sanjeev3 committed:
Update try-review-api-job.md

sanjeev3 committed:
Update try-image-list-api.md

MikeDodaro committed:
using v-gdod in line 11 of metadata

sanjeev3 committed:
Update sdk-and-samples.md

sanjeev3 committed:
Update api-reference.md

sanjeev3 committed:
Update sdk-and-samples.md

sanjeev3 committed:
Update FAQs.md

sanjeev3 committed:
Update FAQs.md

Commit:
update prebuilt domains reference link

PRMerger17 committed:
Merge pull request #29010 from JSeb225/patch-20

Correcting the definition of errors

mmacy committed:
[aks] portal qs edit pass

MikeDodaro committed:
Found a typo

PRMerger18 committed:
Merge pull request #29021 from MGoedtel/Updates1130

updated supported OS's

PRMerger18 committed:
Merge pull request #29027 from DeniseMak/11-30-use-prebuilt-domains-master

update prebuilt domains reference link

mwinkle committed:
Updates to VM sizes

Commit:
Update authoring limits

GitHubber17 committed:
Merge pull request #28969 from MikeDodaro/endpoint-images-summary

Endpoint images summary

Commit:
Merge pull request #29028 from mwinkle/patch-5

Updates to VM sizes

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge pull request #29029 from DeniseMak/11-30-faq-master

Update authoring limits

GitHubber17 committed:
Merge pull request #29001 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-networking.md

chetansriv committed:
Adobe Expereince Manager tutorial

Commit:
Removing duplicate

PRMerger14 committed:
Merge pull request #28968 from MightyPen/patch-1

Rebecca A. suggested this, in earlier PR 28938.

chetansriv committed:
Update Adobe Experience Manager

spelluru committed:
Date

Commit:
Merge pull request #29015 from spelluru/PowerShell1130

Windows PowerShell -> Azure PowerShell

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into PowerShell1130-2

PRMerger20 committed:
Merge pull request #29039 from spelluru/PowerShell1130-2

Date

PRMerger18 committed:
Merge pull request #28991 from hning86/master

stuck on checking exp accts

GitHubber17 committed:
Merge pull request #29026 from sanjeev3/master

Titles and headings

GitHubber17 committed:
Merge pull request #29035 from JSeb225/patch-22

Removing duplicate

Commit:
Update billing-add-change-azure-subscription-administrator.md

Commit:
Update billing-add-change-azure-subscription-administrator.md

Commit:
Update billing-add-change-azure-subscription-administrator.md

Commit:
Update billing-add-change-azure-subscription-administrator.md

PRMerger17 committed:
Merge pull request #29042 from simonxjx/master

Update billing-add-change-azure-subscription-administrator.md

mimig1 committed:
Cosmos DB: Fixed link per verbatim

Commit:
Update iot-hub-arduino-iot-devkit-az3166-get-started.md

PRMerger13 committed:
Merge pull request #29044 from shizn/patch-8

Update iot-hub-arduino-iot-devkit-az3166-get-started.md

neilpeterson committed:
Merge pull request #57 from mmacy/28716-aksportal

[aks] portal qs edit pass

Commit:
Merge pull request #29043 from mimig1/patch-172

Cosmos DB: Fixed link per verbatim

Commit:
making edits after round 1 review

Commit:
merging from origin

Commit:
casing and spelling fixes

Commit:
fixed incorrect transaction limit

linda33wj committed:
Add the missing Parquet format support for PolyBase

linda33wj committed:
Fix Azure Table query samples

PRMerger16 committed:
Merge pull request #29050 from linda33wj/master

Fix ADF copy content

PRMerger18 committed:
Merge pull request #29049 from nstulasi/saneppal-minorfix

fixed incorrect transaction limit

PRMerger13 committed:
Merge pull request #28507 from SHEETHALJS/patch-1

(ForumsTeam) "users" word removed

pareshmu committed:
update neutrona URL

Commit:
Merge pull request #29051 from pareshmu/patch-3

update neutrona URL

yanancai committed:
modify the doc with new feature update

yanancai committed:
fix typos and links

yanancai committed:
fix spelling errors

yanancai committed:
Change Video link and the tips

yanancai committed:
add link for data lake tools for vs

Commit:
Fix to events and fitness

PRMerger16 committed:
Merge pull request #29057 from DeniseMak/12-1-domains-ref-master

Fix to events and fitness

Commit:
Fixes to movie domain

PRMerger12 committed:
Merge pull request #29058 from DeniseMak/12-1-domains-ref-master-movies

Fixes to movie domain

Commit:
minor fixes

Commit:
fixes to reminders and translate

Commit:
freshness

Commit:
Merge pull request #29059 from DeniseMak/12-1-master-translate-reminder

fixes to reminders and translate

Commit:
fixes to tv, weather, and web

PRMerger13 committed:
Merge pull request #29060 from Blackmist/freshness

freshness

cephalin committed:
staged review

Commit:
Merge pull request #29062 from DeniseMak/12-1-master-tv-weather-2

fixes to tv, weather, and web

cephalin committed:
staged review

Commit:
fixes to utilites

PRMerger18 committed:
Merge pull request #29063 from DeniseMak/12-1-master-utilities

fixes to utilites

Commit:
fixes to notes domain

cephalin committed:
staged review

Commit:
Merge pull request #29064 from DeniseMak/12-1-master-notes

fixes to notes domain

Commit:
fixes to Restaurants

cephalin committed:
add FAQ link

Commit:
fix RestaurantReservation domain name to match heading

Commit:
MovieTickets to match heading

PRMerger18 committed:
Merge pull request #29065 from DeniseMak/12-1-master-restaurants

fixes to Restaurants

Commit:
Fixes to music and taxi examples

itechedit committed:
Merge pull request #28957 from cephalin/ssl

use cert article

Commit:
Merge pull request #29067 from DeniseMak/12-1-master-taxi

Fixes to music and taxi examples

itechedit committed:
Merge pull request #28572 from alkohli/SSTOC

Removed Classic node for 8000 series - Go live Dec 1

Commit:
freshness

itechedit committed:
Merge pull request #28623 from MicrosoftDocs/update-assign-user-to-enterprise-app-azure-ad

updating article active-directory-coreapps-assign-user-azure-portal.md

Commit:
updates to device and places

PRMerger18 committed:
Merge pull request #29069 from DeniseMak/12-1-master-device-places-1

updates to device and places

PRMerger12 committed:
Merge pull request #29068 from Blackmist/freshness

freshness

rjagiewich committed:
Merge pull request #28716 from neilpeterson/aks-portal-quickstart

initial file

rjagiewich committed:
Merge pull request #29045 from jeffhollan/func-scale

[Azure Functions] adding more details on scaling

rjagiewich committed:
Merge pull request #29052 from yanancai/master

Updata Azure Data Lake Failed Vertex Debugging Tutorial

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #29071 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #29032 from chetansriv/adobeaem

Adobeaem

mahesh-unnikrishnan committed:
Update active-directory-ds-networking.md

PRMerger15 committed:
Merge pull request #29075 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-networking.md

PRMerger13 committed:
Merge pull request #29061 from hning86/master

minor fixes
